src/api/job.js

```diff
@@ -113,4 +113,22 @@ router.delete('/', async (request, response) => {
   }
 });
 
+router.get('/', async (request, response) => {
+  try {
+    const jobs = await Job.get();
+    if (!jobs) {
+      return response.status(400).json({message: 'Get all of jobs failed!'});
+    }
+
+    return response.status(200).json({
+      message: 'Get all jobs successful, list of jobs is below!',
+      jobs
+    });
+  } catch (error) {
+    console.error(`getJob() >> Error: ${error.stack}`);
+
+    response.status(500).json({message: 'Get all of jobs failed!'});
```

Comment:
`catch (error)` we catch the error but not return to the client, how does the client know the error details if we only return the status code and a hard message like this `response.status(500).json({message: 'Get all of jobs failed!'});`?

Reply:
I thought that users should only know about whether it is succeeded or not, the person who needs more details is the developer. But we have to inform to normal users details of the error either?

Reply:
Fixed.